ledger-core/application/functest/main_func_test.go

```diff
@@ -12,9 +12,19 @@ import (
 	"testing"
 
 	"github.com/insolar/assured-ledger/ledger-core/application/testutils/launchnet"
+	"github.com/insolar/assured-ledger/ledger-core/instrumentation/inslogger/instestlogger"
 )
 
+type testingLogger struct{}
+
+func (t testingLogger) Helper()                {}
+func (t testingLogger) Log(i ...interface{})   {}
+func (t testingLogger) Error(i ...interface{}) {}
+func (t testingLogger) Fatal(i ...interface{}) {}
+
 func TestMain(m *testing.M) {
+	instestlogger.SetTestOutput(testingLogger{})
```

Comment:
Could you explain what is going on here? Do you want to suppress log preparation?

Reply:
there is no testing.T available here

Reply:
had the same issue with network testing suites

Reply:
fixed